app/controllers/shipit/api/base_controller.rb

```diff
@@ -0,0 +1,68 @@
+module Shipit
+  module Api
+    class BaseController < ActionController::Base
+      include Shipit::Engine.routes.url_helpers
```

Comment:
Why this is needed? Should not the controller have access to all url helpers of the engine?